### PR TITLE
OHRI-2109: Add TB Prevention link under Tuberculosis program in the clinical perspectives

### PR DESCRIPTION
**Pull request by lucyjemutai**
## Requirements

Add TB Prevention link under Tuberculosis program in the clinical perspectives
image

### CODE REVIEW

File: packages/esm-tb-app/src/views/dashboard/summary-tiles/tb-prevention-patient-list.component.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { OHRIProgrammeSummaryTiles } from '@ohri/openmrs-esm-ohri-commons-lib';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+function TbPreventionSummaryTiles({ launchWorkSpace }) {
```
Comment (CynthiaKamau) [inline packages/esm-tb-app/src/views/dashboard/summary-tiles/tb-prevention-patient-list.component.tsx:5]: The `launchWorkspace` prop is unused

File: packages/esm-tb-app/src/views/dashboard/patient-list-tabs/tb-prevention-patient-list.component.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tabs, Tab, TabPanels, TabPanel, TabList } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+
+interface OverviewListProps {
+  patientUuid: string;
+}
+
+const TbPreventionPatientListTabs: React.FC<OverviewListProps> = ({ patientUuid }) => {
```
Comment (CynthiaKamau) [inline packages/esm-tb-app/src/views/dashboard/patient-list-tabs/tb-prevention-patient-list.component.tsx:9]: The `patientUuid` prop is unused

**Review by CynthiaKamau — COMMENTED** [review]
Thanks @lucyjemutai , please fix the conflicts

**Comment by lucyjemutai** [comment]
@CynthiaKamau , Thank you for the review,  I have closed this pr and created another with the issues addressed. thanks
[](https://github.com/UCSF-IGHS/openmrs-esm-ohri/pull/1782)

**Pull request closed without merging**